docker/actions/kind.go

```diff
@@ -274,40 +273,3 @@ func image(version string) string {
 	}
 	return defaults.Image
 }
-
-func removeExitedELBWithNameConflict(name string) error {
```

Comment:
This wasn't used?

Reply:
It was used in create ELB, but I don't think we should keep this logic at this stage.
Eventually, we can add it back later...

Reply:
sgtm